packages/events-example/README.md

```diff
@@ -1,59 +1,115 @@
 
-reconcile nile/pulumi deploys
+```json
+{
+  "greeting": "Come with me if you want to live."
+}
+```
+
+## Configure Your Data Plane ##
+
+You can follow the steps to [create a Pulumi project](https://www.pulumi.com/docs/get-started/aws/create-project/)
+and [deploy the stack](https://www.pulumi.com/docs/get-started/aws/deploy-stack/).
+
+More concisely, in a separate directory:
 
+```bash
+$ mkdir nile-examples && cd nile-examples
+$ pulumi new aws-typescript
 ```
-USAGE
-  $ nile reconcile [--json] [--basePath <value>] [--workspace <value>] [--email <value>] [--password <value>]
-    [--organization <value>] [--entity <value>] [-s] [--region <value>]
-
-FLAGS
-  -s, --status            check current status of your control and data planes
-  --basePath=<value>      [default: http://localhost:8080] root URL for the Nile API
-  --email=<value>         [default: developer@demo.com] developer email address
-  --entity=<value>        an entity type in your Nile workspace
-  --organization=<value>  an organization in your Nile workspace
-  --password=<value>      [default: very_secret] developer password
-  --region=<value>        [default: us-west-2] AWS region
-  --workspace=<value>     [default: dev] your Nile workspace name
-
-GLOBAL FLAGS
-  --json  Format output as json.
-
-DESCRIPTION
-  reconcile nile/pulumi deploys
+
+and accept the defaults:
+
 ```
-<!-- commandsstop -->
+project name: (nile-examples) 
+project description: (A minimal AWS TypeScript Pulumi program) 
+Created project 'nile-examples'
+
+Please enter your desired stack name.
+To create a stack in an organization, use the format <org-name>/<stack-name> (e.g. `acmecorp/dev`).
+stack name: (dev) 
+Created stack 'dev'
+
+aws:region: The AWS region to deploy into: (us-east-1)
+Saved config
+```
+
+Run `pulumi up` to ensure that you've configured Pulumi correctly. This will
+create a stack named `dev`. We won't be using this stack, but its presence
+verifies that you're ready to proceed.
+
+## Create and Execute a Nile Agent ##
+
+In the `events-example` package root, run `yarn install && yarn build` to
+create your custom Nile agent binaries.
+
+To run the binary, you'll need to pass arguments for your Nile workspace,
+organization, and entity type, plus your developer login. The workspace
+and entity type are identified by name, but the organization requires an id.
+
+| The organization id is not visible in the NAD yet, but can be obtained from
+| the URL when you select an org. In this case:
+|
+| `https://nad.thenile.dev/cyberdyne/organization/org_02qfJTCBve6bw0XlxC92CG`
+|
+| the organization id is `org_02qfJTCBve6bw0XlxC92CG`.
+
+Execute the binary with:
+
+```bash
+./bin/dev reconcile --basePath https://prod.thenile.dev \
+--workspace YOUR_WORKSPACE \
+--organization YOUR_ORG \
+--entity SkyNet \
+--email YOUR_EMAIL \
+--password YOUR_PASSWORD
+```
+
+This should synchronize your Pulumi stack and instantiate the SkyNet instance
```

Comment:
nit: will! it WILL synchronize your Pulumi stack

Reply:
As long as Pulumi isn't finnicky... Of _course_ our stuff will work. :)